### ISSUE TITLE
Friday night funkin' Roses+Thorns retrospecter remix kade engine+a little sneakpeak

### ISSUE THREAD

**Issue by Ali109956** [luckydog7/Funkin-android#554]
Download link:
https://www.mediafire.com/download/08v6ru4v1smm4b6
Original mod:
https://gamebanana.com/mods/44266
Kade engine android port:
https://github.com/KlavierGayming/K-Dev-Android
https://youtu.be/dTE0hcRGjYg
Screenshots:
![Screenshot_20210429_224455_com ninjamuffinkade funkin](https://user-images.githubusercontent.com/83115279/116609462-124bf880-a93d-11eb-92bf-469f2eac3678.jpg)
![Screenshot_20210429_224629_com ninjamuffinkade funkin](https://user-images.githubusercontent.com/83115279/116609483-19730680-a93d-11eb-9f1a-0f574398196d.jpg)
![Screenshot_20210429_224747_com ninjamuffinkade funkin](https://user-images.githubusercontent.com/83115279/116609501-1d9f2400-a93d-11eb-9e62-44f3cd6e1657.jpg)
![Screenshot_20210429_224757_com ninjamuffinkade funkin](https://user-images.githubusercontent.com/83115279/116609516-21cb4180-a93d-11eb-8d21-1f662997232f.jpg)

Sneakpeak:
![Screenshot_20210429_224310_com ninjamuffinkade funkin](https://user-images.githubusercontent.com/83115279/116609545-2c85d680-a93d-11eb-9e2e-94050bd11f5d.jpg)

**Comment by ALAE1234M** [comment]
I like this buttons can you do it

**Comment by netonero** [comment]
Cant install it
What did you do

**Comment by ROBIKALIVE13** [comment]
> Cant install it
> What did you do

it can be storage problem

**Comment by Ali109956** [comment]
> > Cant install it
> > What did you do
> 
> it can be storage problem



Yes it is a storage problem since for some reason the file is 205mb instead of 134mb to 168mb size

**Comment by netonero** [comment]
![Screenshot_20210430_164916](https://user-images.githubusercontent.com/80380726/116694751-3c132700-a9f2-11eb-93fa-b9180cdfb630.jpg)
Yeah

**Comment by Ali109956** [comment]
> ![Screenshot_20210430_164916](https://user-images.githubusercontent.com/80380726/116694751-3c132700-a9f2-11eb-93fa-b9180cdfb630.jpg)
> Yeah


I know how to fix this just download the file again than install it

**Comment by netonero** [comment]
> > ![Screenshot_20210430_164916](https://user-images.githubusercontent.com/80380726/116694751-3c132700-a9f2-11eb-93fa-b9180cdfb630.jpg)
> > Yeah
> 
> I know how to fix this just download the file again than install it

![Screenshot_20210501_051649](https://user-images.githubusercontent.com/80380726/116755147-8faa6280-aa3c-11eb-9abf-eaf30012e45c.jpg)
This is the third attempt of me downloading the same apk again
